src/Analysis/GWASResults/Views/Results/ResultsPheWeb/TopLociTable/TopLociTable.jsx

```diff
@@ -0,0 +1,187 @@
+      });
+    }
+    // When the user updates sorting set page to first page
+    return setLociTableState({
+      ...lociTableState,
+      currentPage: 1,
+    });
+  };
+
+  const columns = [
+    {
+      title: 'Variant',
+      dataIndex: 'variant',
+      key: 'variant',
+      sorter: (a, b) => a.variant.localeCompare(b.variant),
+      children: [
+        {
+          title: (
+            <Input
+              placeholder='Search by Variant'
+              value={lociTableState.variantSearchTerm}
+              onChange={(event) => handleSearchTermChange(event, 'variant')}
+              suffix={<SearchOutlined />}
+            />
+          ),
+          dataIndex: 'variant',
+        },
+      ],
+    },
+    {
+      title: 'Nearest Gene(s)',
+      dataIndex: 'nearest_genes',
+      key: 'nearest_genes',
+      sorter: (a, b) => a.nearest_genes.localeCompare(b.nearest_genes),
+      children: [
+        {
+          title: (
+            <Input
+              placeholder='Search by Nearest gene(s)'
+              suffix={<SearchOutlined />}
+              value={lociTableState.nearestGenesSearchTerm}
+              onChange={(event) => handleSearchTermChange(event, 'nearest_genes')}
+            />
+          ),
+          dataIndex: 'nearest_genes',
+        },
+      ],
+    },
+    {
+      title: 'AF',
+      dataIndex: 'af',
+      key: 'af',
+      sorter: (a, b) => a.af.toString().localeCompare(b.af.toString()),
+
+      children: [
+        {
+          title: (
+            <Input
+              placeholder='Search by Af'
+              suffix={<SearchOutlined />}
+              value={lociTableState.afSearchTerm}
+              onChange={(event) => handleSearchTermChange(event, 'af')}
+            />
+          ),
+          dataIndex: 'af',
+        },
+      ],
+    },
+    {
+      title: 'P-value',
+      dataIndex: 'pval',
+      key: 'pval',
+      sorter: (a, b) => a.pval.toString().localeCompare(b.pval.toString()),
+      children: [
+        {
+          title: (
+            <Input
+              placeholder='Search by P-value'
+              suffix={<SearchOutlined />}
+              value={lociTableState.pvalSearchTerm}
+              onChange={(event) => handleSearchTermChange(event, 'pval')}
+            />
+          ),
+          dataIndex: 'pval',
+        },
+      ],
+    },
+  ];
+
+  return (
+    <section className='top-loci'>
+      <h2>Top Loci</h2>
+      <div className='table-header'>
+        <Button onClick={() => downloadTSVFromJson('topLociTSV.tsv', data)}>
+          Download All Results
+        </Button>
+      </div>
+      <Table
+        dataSource={filteredData}
+        columns={columns}
+        rowKey={(record) => record.pos}
+        onChange={handleTableChange}
+        pagination={{
+          current: lociTableState.currentPage,
+          defaultPageSize: 10,
+          showSizeChanger: true,
+          pageSizeOptions: ['10', '20', '30'],
```

Comment:
I think ['10', '50', '100', '500', '1000'] can make this UI more useful.

Reply:
Great idea. This has been updated to: 

```  pageSizeOptions: ['10', '50', '100','500','1000'], ```